documentation/src/main/asciidoc/public/reference/_backend-elasticsearch.adoc

```diff
@@ -73,6 +76,38 @@ and as a result <<mapper-orm-schema-management-manager-create-or-update,automati
 (<<schema-management-concepts-update-failure,not recommended in production>>)
 <<schema-management-concepts-index-closing,will fail when trying to update analyzer definitions>>.
 
+[[backend-elasticsearch-compatibility-amazon-opensearch-serverless]]
+=== Amazon OpenSearch Serverless (incubating)
+
+include::../components/_incubating-warning.adoc[]
+
+link:{amazonOpenSearchServerlessUrl}[Amazon OpenSearch Serverless] compatibility is implemented and incubating;
+feel free to provide feedback on https://hibernate.atlassian.net/browse/HSEARCH-4867[HSEARCH-4867].
+
+However, be aware that:
+
+* Hibernate Search does not currently get tested against Amazon OpenSearch Serverless;
+see https://hibernate.atlassian.net/browse/HSEARCH-4919[HSEARCH-4919].
+* Connecting to an Amazon OpenSearch Service cluster
+requires <<backend-elasticsearch-configuration-aws,proprietary authentication>> that involves extra configuration.
+* Compatibility with Amazon OpenSearch Service must be enabled
```

Comment:
```suggestion
* Compatibility with Amazon OpenSearch Serverless must be enabled
```
Maybe say that it's about serverless specifically (and in the previous line item too)

Reply:
Good catch, will do